docs/logs/getting-started/getting-started.csproj

```diff
@@ -3,12 +3,7 @@
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Exporter.Console\OpenTelemetry.Exporter.Console.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.1" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.1.1" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp2.1'">
+  <ItemGroup>
```

Comment:
why is 2.1 being removed?

Reply:
We used to have different code for 2.1 which used ServiceCollection instead of LoggerFactory. We could use LoggerFactory for 2.1 as well. That way the code is same for all the  target frameworks too.

Reply:
The LoggerFactory.Create() method does not exist in .Net Core 2.1 (i.e in Msft.Ext.Logging version 2.1). Only option is to use the ServiceCollection method!
Its an unfortunate and ugly thing, but there is no way out of it until Aug 2021, when 2.1 is officially going out of support!